### DISCUSSION TITLE
How to view line attribution for inlined functions in perfview?

### DISCUSSION THREAD

**Discussion by anter1** [microsoft/perfview#2125] [Q&A]
Let's say that in C++ I have function A in fileA.cpp which calls function B in fileB.cpp. Let's say function B is inlined into A by the compiler/linker.

When investigating a CPU sampling profile I can use alt+D or right click->goto source and it will show me line attribution for CPU usage of function A by opening fileA.cpp. In my case a lot of the CPU usage comes from function B. How can I see line attribution from perfview for the inlined function B in fileB.cpp? I don't seem to find a way to make perfview open fileB.cpp when function A is defined in fileA.cpp

Thanks.

**Comment by cincuranet** [comment]
@brianrob I did a quick research and it looks like PDB contains inlining information. Could we maybe use that information here?
